doc/doc.css

```diff
@@ -1,44 +1,226 @@
-.hero{
-    padding: 4rem;
+
+
+/* -------------   */
+
+@import url(../styles/main.css);
+
+
+/* ***** page layout *** */
+
+.page-layout{
     display: grid;
-    grid-template-columns: 6rem 1fr 1fr 2rem;
+    grid-template-columns: 1fr 3fr 0.3fr;
     grid-template-areas: 
-    ". left right .";
-    /* height: 80vh; */
-gap: 1rem;
+    "header  header header"
+    "sidebar  main   .";
+}
+
+
+/* ***** header ****** */
 
+.header{
+    grid-area: header;
+    padding: 1rem 2.8rem;
+    display: flex;
+    position: sticky;
+    top: 0;
+    z-index: 3;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0 10px 8px -8px #d6d6d6;
+    font-weight: 500;
+    background-color: var(--primary-color)
+}
 
+.header__logo{
+    display: flex;
+    margin-left: 7rem;
+}
+.header__logo > img{
+    width: 15rem;
+    height: 5.8rem
 }
 
-.hero > div {
-    padding: 1rem;
+.header__links{
+    list-style: none;
+    display: flex;
+    font-size: 2.9rem;
+    align-items: center;
+    margin-right: 6 rem;
+}
+
+.header__links li i{
+    color: black;
+}
 
+.header__links li i:hover{
+    cursor: pointer;
+    /* color: var(--primary-dark-color) */
+    color: var(--secondary-color)
 }
 
-.hero-left{
-    grid-area: left;
+
+/* ****** Sidebar ******* */
+
+.sidebar-main{
+    position: relative;
+    grid-area: sidebar;
+    
+}
+
+.sidebar{
+    padding: 3rem;
+    padding-left: 4.5rem;
+    padding-right: 3rem;
+    position: sticky;
+    overflow-y: auto;
+    top: 7.5rem;
```

Comment:
top property doesn't work for static positioning, you can remove it

Reply:
It is creating problem after removing "top". That's why I have used it.